.github/workflows/issue_event_moves_card.yml

```diff
@@ -0,0 +1,126 @@
+        type: string
+        default: 'NONE'
+        description: 'The priority of the issue'
+      WORKLOAD:
+        type: string
+        default: 'NONE'
+        description: 'The workload of the issue'
+    secrets:
+        PAT:
+          required: true
+
+jobs:
+  Move_card:
+    runs-on: ubuntu-latest
+    steps:
+    - name: "Move card in projects"
+      id: move_cards
+      run: |
+        ISSUE_NODE_ID="${{ inputs.ISSUE_NODE_ID }}"
+        PRIORITY="${{ inputs.PRIORITY }}"
+        WORKLOAD="${{ inputs.WORKLOAD }}"
+        COLUMN_NAME="${{ inputs.COLUMN_NAME }}"
+        CURRENT_COLUMN_NAME="${{ inputs.CURRENT_COLUMN_NAME }}"
+        echo "ISSUE_NODE_ID=$ISSUE_NODE_ID"
+        echo "PRIORITY=$PRIORITY"
+        echo "WORKLOAD=$WORKLOAD"
+        echo "COLUMN_NAME=$COLUMN_NAME"
+        echo "CURRENT_COLUMN_NAME=$CURRENT_COLUMN_NAME"
+        # Get the projects associated with the organization
+        projects=$(curl --request POST \
+            --url https://api.github.com/graphql \
+            --header "Authorization: Bearer ${{ secrets.PAT }}" \
+            --data '{"query":"{organization(login: \"DLR-AMR\") {projectsV2(first: 20) {nodes {id title}}}}"}')
+        echo "projects=$projects"
+        # Get the project IDs
+        project_ids=$(echo $projects | jq -r '.data.organization.projectsV2.nodes[].id')
+        for project_id in $project_ids; do
+          echo "project_id is $project_id"
+          # Get the issues within the project
+          project_fields_and_issues=$(curl --request POST \
+            --url https://api.github.com/graphql \
+            --header "Authorization: Bearer ${{ secrets.PAT }}" \
+            --data "{\"query\":\"{ node(id: \\\"$project_id\\\") { ... on ProjectV2 { items(first: 100) { nodes { id fieldValues(first: 8) { nodes { ... on ProjectV2ItemFieldTextValue { text field { ... on ProjectV2FieldCommon { name } } } ... on ProjectV2ItemFieldDateValue { date field { ... on ProjectV2FieldCommon { name } } } ... on ProjectV2ItemFieldSingleSelectValue { name field { ... on ProjectV2FieldCommon { name } } } } } content { ... on Issue { id } } } } fields(first: 20) { nodes { ... on ProjectV2SingleSelectField { id name options { id name } } } } } } }\"}")
+          echo "issues=$project_fields_and_issues"
+          # Extract the issue ids and PVII ids. The issue_id is the global_id, the PVII id is the project specific id
+          issue_ids=$(echo $project_fields_and_issues | jq -r '.data.node.items.nodes[].content.id')
+          PVII_ids=$(echo $project_fields_and_issues | jq -r '.data.node.items.nodes[].id')
+          # In the following loop we need both values, so we combine them into a single string
+          combined_ids=$(paste -d, <(echo "$issue_ids") <(echo "$PVII_ids"))
+          for combined_id in $combined_ids; do
+            issue_id=$(echo $combined_id | cut -d, -f1)
+            PVII_id=$(echo $combined_id | cut -d, -f2)
+            echo "PVII_id is $PVII_id"
```

Comment:
I think we should also delete this line as the PVII_id is printed for each issue related to the project

Reply:
ok